### ISSUE TITLE
Calling Rails Util Methods

### ISSUE THREAD

**Issue by dschadd** [LaunchPadLab/rails_util#37]
Currently, we are unable to directly call methods in Rails Util. Calling methods requires explicitly declaring the module `RailsUtil` and the module containing the required method. For example you cannot directly call `convert_timezone`, but must declare `RailsUtil::TimezoneHelper.convert_timezone`. I would like to change that so all methods contained in the gem are able to be imported or just simply called.

Also, the documentation uses `yard` which I had trouble using with Rails 2.7. I would like to extract the documentation, update/create it, and place it in the Github repo Readme.

**Comment by inveterateliterate** [comment]
the main issue lgtm. I'm all for doc updates too, but let's capture that in a separate issue incase multiple team members want to get involved.

**Comment by chawes13** [comment]
@dschadd Is 2.7 your Rails version or your Ruby version? 

Are there any alternatives to Yard in the Ruby ecosystem? We have autogenerated docs in our front-end libraries and I personally really like them. Maybe an approach we could take is having a README that links to a `docs.md` file that is autogenerated (e.g., https://github.com/LaunchPadLab/lp-components)?

**Comment by chawes13** [comment]
And forgive my ignorance here, but is this an issue that is addressable in this repo or would we need something like the below in our Rails apps? Maybe both?

```ruby
class ApplicationHelper
  RailsUtil.constants.each do |s|
    m = RailsUtil.const_get(s)
    if(m.instance_of?(Module))
      include m
    end
  end
end
```

**Comment by inveterateliterate** [comment]
My hunch says we're just missing some gem configurations (like having some kind of Core class?). Truthfully this repo is less of a true gem and more of a collection of modules. We have wanted to pursue packaging it more nicely before but have not yet prioritized it. I think that makes it a good candidate for this hackfest. If that ends up not being the case then we certainly explore a Rails template helper like the above!

Token master might be a relatively straight forward example to reference: https://github.com/LaunchPadLab/token-master/blob/master/lib/token_master/core.rb

Happy to pair with whoever wants to work on it.

**Comment by dschadd** [comment]
> @dschadd Is 2.7 your Rails version or your Ruby version?
 Yes it is it was complaining about not having 2.4 on my machine.
